addons/website_slides_survey/models/slide_slide.py

```diff
@@ -65,6 +65,13 @@ def _compute_is_preview(self):
             if slide.slide_category == 'certification' or not slide.is_preview:
                 slide.is_preview = False
 
+    @api.depends('slide_type')
+    def _compute_slide_icon(self):
+        super(Slide, self)._compute_slide_icon()
+        for slide in self:
+            if slide.slide_type == 'certification':
+                slide.slide_icon = 'fa-trophy'
```

Comment:
no need to iterate 2 times on self
```suggestion
        certification = self.filtered(lambda slide: slide.slide_type == 'certification')
        certification.slide_icon = 'fa-trophy'
        super(Slide, self - certification)._compute_slide_icon()
```